specs2/BUILD

```diff
@@ -4,11 +4,11 @@ java_import(
     name = "specs2",
     jars = [],
     exports = [
-        "@io_bazel_rules_scala_org_scalaz_scalaz_core//jar",
-        "@io_bazel_rules_scala_org_scalaz_scalaz_effect//jar",
-        "@io_bazel_rules_scala_org_specs2_specs2_common//jar",
-        "@io_bazel_rules_scala_org_specs2_specs2_core//jar",
-        "@io_bazel_rules_scala_org_specs2_specs2_matcher//jar",
+        "@io_bazel_rules_scala_org_scalaz_scalaz_core",
```

Comment:
this looks weird to me... I guess it is the default target somehow.

Reply:
`scala_maven_import_external` creates the target `@io_bazel_rules_scala_org_scalaz_scalaz_core:io_bazel_rules_scala_org_scalaz_scalaz_core` and an alias to `@io_bazel_rules_scala_org_scalaz_scalaz_core//jar:jar`, I can add the full path